sdk/azcore/runtime/policy_include_response_test.go

```diff
@@ -0,0 +1,41 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package runtime
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRawResponsePolicy(t *testing.T) {
+	srv, close := mock.NewServer()
+	defer close()
+	// add a generic HTTP 200 response
+	srv.SetResponse()
```

Comment:
It would be good to also verify the policy provides the final response after polling or retrying

Reply:
Good idea, fixed.